### ISSUE TITLE
Installing this on top of a blank `starter` or `starter_dev` site results in an error

### ISSUE THREAD

**Issue by noahwsmith** [Islandora/islandora_mirador#11]
**What steps does it take to reproduce the issue?**

```
git clone git@github.com:Islandora-Devops/isle-dc.git
cd isle-dc/
make starter
docker-compose exec drupal bash -c "chown -R nginx:nginx web"
docker-compose exec drupal bash -c "chmod u+w web/sites/default/settings.php"
docker-compose exec -u nginx drupal bash -c "composer require islandora/islandora_mirador"
docker-compose exec -u nginx drupal bash -c "drush en islandora_mirador"
```

* What happens?

```
In UnmetDependenciesException.php line 100:

  Configuration objects provided by <em class="placeholder">islandora_mirador</em> have unmet dependencies: <em class="placeholder">core.entity_view_display.node.islandora_object.mirador (islandora_defaul
  ts)</em>

```

* What did you expect to happen?

I thought `islandora_defaults` was in conflict with this module (see https://github.com/Islandora-Devops/islandora_install_profile_demo/issues/28#issuecomment-1297440987) and `islandora_defaults` is not listed as a dependency here, so I think the config dependency should probably be be removed or edited. 

**Which version of Islandora are you using?**

2.4.3

**Comment by noahwsmith** [comment]
oh @ajstanley I think your #9 fixed this, but the default install relies on tags to include the latest. Tagging a 2.2.2 on this repo should fix this issue for other folks. I'll switch to `2.x-dev` for my work in the meanwhile...

**Comment by noahwsmith** [comment]
Yep, that does it:

```
noahs-m2-mbp-4:isle-dc noah$ docker-compose exec -u nginx drupal bash -c "composer require islandora/islandora_mirador:2.x-dev"
Info from https://repo.packagist.org: #StandWithUkraine
./composer.json has been updated
Running composer update islandora/islandora_mirador
Loading composer repositories with package information
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies
Lock file operations: 0 installs, 1 update, 0 removals
  - Upgrading islandora/islandora_mirador (2.2.1 => 2.x-dev ee0a777)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
  - Syncing islandora/islandora_mirador (2.x-dev ee0a777) into cache
  - Removing islandora/islandora_mirador (2.2.1)
  - Installing islandora/islandora_mirador (2.x-dev ee0a777): Cloning ee0a777089 from cache
Package doctrine/reflection is abandoned, you should avoid using it. Use roave/better-reflection instead.
Package silex/silex is abandoned, you should avoid using it. Use symfony/flex instead.
Package symfony/debug is abandoned, you should avoid using it. Use symfony/error-handler instead.
Package webmozart/path-util is abandoned, you should avoid using it. Use symfony/filesystem instead.
Generating autoload files
68 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
noahs-m2-mbp-4:isle-dc noah$ docker-compose exec -u nginx drupal bash -c "drush en islandora_mirador"
 [success] Successfully enabled: islandora_mirador
noahs-m2-mbp-4:isle-dc noah$
```

**Comment by rosiel** [comment]
[tag 2.2.2 created](https://github.com/Islandora/islandora_mirador/releases/tag/2.2.2).

**Comment by rosiel** [comment]
Can you test whether this is still an issue?

**Comment by rosiel** [comment]
* The config in question was [moved to /optional](https://github.com/Islandora/islandora_mirador/commit/ee0a77708951174aab8345b0746c77d2ea2e62f7) back on Oct 26, so this should no longer block installation.
* However you might not have gotten the most recent version because the 2.2.2 tag was only cut 2 weeks ago and when you `composer require`'d mirador you didn't state a version. Perhaps you got the latest tag without this improvement?
* When I installed islandora_mirador:^2.2.2 on top of a starter_dev playbook, it installed fine. 
* However I did notice that config depends on islandora_defaults so i'm making a PR to take it out.
